### ISSUE TITLE
Question: how to set new value to configuration parameters

### ISSUE THREAD

**Issue by rockstar2020** [exislow/tidal-dl-ng#5]
Hello,
I'm struggling to pass new values of configuration paramaters via CLI.
Following your instructions I tried every possible methods I could think of but couldn't pass a new value to certain parameter.
Please see one of the ways I tried to pass the respective arguments to set a new value to download folder parameter.
Can you provide an example to pass multiple values for multiple parametrs in one line?
Also, is it possible to combine dl command and cfg in one CLI command?

Many thanks in advance. 

**The CLI command:**
tidal-dl-ng cfg --names download_base_path "/mnt/test/"

**Error:**
Got unexpected extra argument (/mnt/test/)

**Comment by exislow** [comment]
This should work: 

tidal-dl-ng cfg download_base_path "/mnt/test/"

I will extend the readme soon. Otherwise you could simples edit the config file (~/.tidal-dl-ng_settings.json) with and editor like vim.

**Comment by rockstar2020** [comment]
Thanks!
That was the first option I tried but it doesn't work!

I tried the following again just now to be 100% sure:
tidal-dl-ng cfg download_base_path "/mnt/test/"

and then I got the following error:
Got unexpected extra arguments (download_base_path /mnt/test/)

Of course I did use nano to edit the file for testing purposes, but I have lot of automtations where I use CLI to download various qualities into desired download folder. I'm currently using original tidal-dl by Yaronzz to do this.

Also, I wanted to pick your brain as I'm puzzled how I can get Hi-Res/MQA quality downloaded.
The thing is sometimes ago I was able to download Master Quality but nowadays it's hit or miss. I can't figure out why!

Below is the snapshot of flac properties I downloaded few weeks ago:
![image](https://github.com/exislow/tidal-dl-ng/assets/35903046/11f83f3e-2c5f-4248-b2a3-d54361eecca0)

and this is the properties of the same file downloaded with exactly the same parameters on the same machine with the same account:
![image](https://github.com/exislow/tidal-dl-ng/assets/35903046/440bf021-8478-4fe1-b2bf-983a5d1cda22)


Any thoughts? I can't think of anything else that causes this limitation.
In fact, I've been using Yaronzz's tidal-dl for the past month or so and then I came across your project in my search to find a better and more updated tool to get me a higher download quality.

Thanks
